sdk/python/tests/integration/feature_repos/universal/data_source_creator.py

```diff
@@ -17,11 +16,7 @@ def __init__(self, project_name: str, *args, **kwargs):
     def create_data_source(
         self,
         df: pd.DataFrame,
-        destination_name: str,
-        event_timestamp_column="ts",
-        created_timestamp_column="created_ts",
-        field_mapping: Optional[Dict[str, str]] = None,
-        timestamp_field: Optional[str] = None,
+        **kwargs,
     ) -> DataSource:
```

Comment:
functionality change: due to create_data_source needing an indeterminate number of arguments. The base class should accept kwargs and the concrete class should then process user arguments on whether sufficient inputs were given.